packages/@okta/vuepress-site/docs/guides/custom-url-domain/enable-the-custom-domain/index.md

```diff
@@ -10,18 +10,25 @@ Use the configuration wizard to walk through the steps to customize your Okta UR
     * The **Update Certification** button appears if a custom URL domain is already configured for your org. To delete the current configuration, click **Restore to default**.
 
 ### Add your subdomain information
-On the Domain page of the configuration wizard, enter your subdomain name, for example, `login.example.com`, and then click **Next**. Verifying domain ownership is the next step in the configuration wizard.
+On the Domain page of the configuration wizard, enter your subdomain name, for example, `login.exampledomain.com`, and then click **Next**. Verifying domain ownership is the next step in the configuration wizard.
 
 ### Create a DNS TXT record
 You need to add a DNS TXT record to your domain to verify ownership of your domain with Okta before Okta can serve traffic over it. This record includes the Okta-generated values provided in the **Host** and **Data** columns of the table on the **Verify domain ownership page**. Okta verifies that you own your domain when it finds the TXT record that contains the generated value.
 
 1. On the **DNS Records** page of the configuration wizard, copy the values of the **Host** and **Data** columns into a text file.
+
 2. Sign in to your Domain Name registrar and locate the option to modify your DNS records.
+
 3. Add a TXT record and paste the value that you copied from the **Host** column into the appropriate field, for example, the **Name** or **Host** field.
-> Important: Depending on your registrar, you may only need to enter `_oktaverification` rather than `_oktaverification.login.example.com`. If your registrar doesn't support the value that you enter, verification fails and your custom URL domain configuration is incomplete.
+
+> **Note**: Depending on your registrar, you may only need to enter `_oktaverification` rather than `_oktaverification.login.exampledomain.com`. If your registrar doesn't support the value that you enter, verification fails and your custom URL domain configuration is incomplete.
```

Comment:
> **Note**: Depending on your registrar, you may only need to enter `_oktaverification` rather than `_oktaverification.login.example.com`. If your registrar doesn't support the value that you enter, verification will fail and your custom URL domain configuration will be incomplete. Perform a DNS lookup of your `_oktaverification` DNS record to verify that it's correctly configured. To do this on Linux or macOS, use the `dig` command. For example: `dig _oktaverification.login.example.com`

Reply:
sigh - this was my fault.  I did it right when I initially created this guide with `example.com`, and yet somehow had a momentary lapse of reason yesterday afternoon. I'll fix them. :(